.changeset/calm-prompts-resolve.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/editor': patch
+---
+
+Fixed version-specific reads across editor namespaces and prevented prompt draft updates from changing cached published reads.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)